### DISCUSSION TITLE
Recommendation: gem5-slicc VS Code Extension

### DISCUSSION THREAD

**Discussion by ImagineBoom** [gem5/gem5#480] [gem5-dev]
Hello everyone, let me recommend a VS Code extension:  [`gem5-slicc` ](https://marketplace.visualstudio.com/items?itemName=tsinghua-hexin-joint-institute.gem5-slicc)

Empower your gem5 SLICC coding experience with our VS Code extension – alleviate your eye strain!

Eagerly anticipating everyone's usage and welcoming feedback.

**Comment by griffin-warrior** [comment]
Good job!

**Comment by powerjg** [comment]
Very cool! Thanks for the contribution!
